### ISSUE TITLE
Browser demo

### ISSUE THREAD

**Issue by hrj** [danfickle/neoflyingsaucer#1]
Is it possible to have the browser demo in this fork of the project?

**Comment by danfickle** [comment]
Thanks a lot for your interest and help with this project. I take it you found the browser demo. However, for others looking it is at:
/flying-saucer-examples/src/main/java/org/xhtmlrenderer/demo/browser/BrowserStartup.java

Out of interest, do you intend on using the swing component? I was intending on focusing on the Java2D renderer and PDF, but if people are using the swing components I can keep an eye on them too.

The browser works on some sites but not others. For example http://getbootstrap.com/css roughly works but not http://github.com

Again, thanks for your help!

**Comment by hrj** [comment]
Yes, I noticed the demo just about the time I saw your comment :)

I am indeed planning to use the swing component for a little browser I am building. I have been experimenting with a lot of libraries in the past few weeks, and so far flyingsaucer looks good, so I will probably continue to use it. I also like your fork, because it's pure Java.

About the demo browser in this code, I am not able to use it for websites yet; only the demo links work. Am investigating further.
